### PR TITLE
[fix] fix some problem

### PR DESCRIPTION
**Pull request by 133tosakarin**
This pr changes some init size for array

### CODE REVIEW

**Comment by 133tosakarin** [comment]
run buildall

File: iotdb-core/datanode/src/main/java/org/apache/iotdb/db/storageengine/dataregion/memtable/AlignedWritableMemChunk.java [hunk 1]
```diff
@@ -319,7 +321,30 @@ public void removeColumn(String measurementId) {
...
+  public Object newArrayByType(TSDataType type, int length) {
+    switch (type) {
+      case BOOLEAN:
+        return new boolean[length];
+      case INT32:
+      case DATE:
+        return new int[length];
+      case INT64:
+      case TIMESTAMP:
+        return new long[length];
```
Comment (OneSizeFitsQuorum) [inline iotdb-core/datanode/src/main/java/org/apache/iotdb/db/storageengine/dataregion/memtable/AlignedWritableMemChunk.java:336]: revert

File: iotdb-core/datanode/src/main/java/org/apache/iotdb/db/storageengine/dataregion/memtable/WritableMemChunk.java [hunk 2]
```diff
@@ -326,16 +327,77 @@ public String toString() {
...
+    switch (type) {
+      case BOOLEAN:
+        return new boolean[length];
+      case INT32:
+      case DATE:
+        return new int[length];
+      case INT64:
+      case TIMESTAMP:
+        return new long[length];
+      case FLOAT:
```
Comment (OneSizeFitsQuorum) [inline iotdb-core/datanode/src/main/java/org/apache/iotdb/db/storageengine/dataregion/memtable/WritableMemChunk.java:340]: revert

File: iotdb-core/datanode/src/main/java/org/apache/iotdb/db/utils/datastructure/TimSort.java [hunk 3]
```diff
@@ -65,9 +68,22 @@ default void sort(int lo, int hi) {
...
+    if (hi - lo <= 1000) {
+      sort(lo, mid);
+      sort(mid, hi);
+      merge(lo, mid, hi);
+      return;
+    }
+    CompletableFuture<?> future = CompletableFuture.runAsync(() -> sort(lo, mid));
     sort(mid, hi);
-    merge(lo, mid, hi);
+    try {
```
Comment (OneSizeFitsQuorum) [inline iotdb-core/datanode/src/main/java/org/apache/iotdb/db/utils/datastructure/TimSort.java:79]: revert

**Pull request closed without merging**